core/src/main/java/com/linecorp/armeria/common/logging/RequestContextExporter.java

```diff
@@ -140,6 +140,9 @@ public static RequestContextExporterBuilder builder() {
      */
     public boolean containsAttribute(AttributeKey<?> key) {
         requireNonNull(key, "key");
+        if (attrs == null) {
+            return false;
+        }
         return Arrays.stream(attrs).anyMatch(e -> e.key.equals(key));
```

Comment:
It's unrelated to this PR, but I noticed these methods seem fairly slow. I think the exporter is generally a singleton so object allocation shouldn't matter so much - should we use maps instead of arrays here?

Reply:
I thought this method is used only for checking how the exporter is configured. What's your use case?

Reply:
Not sure ;) But since these are public methods, figured it's worth considering it since doesn't seem that complicated to speed them up.